app/views/layouts/base.html.slim

```diff
@@ -14,5 +14,5 @@ html lang="en-US"
     = yield :styles
 
   body
-    = render "application/alerts"
+    = content_for?(:alerts) ? render(inline: '') : render('application/alerts')
```

Comment:
Switching the `if` to `unless` would make it do the opposite thing, in this case, render the alerts unless they are already defined.